source/command-line-options.txt

```diff
@@ -33,18 +35,77 @@ Use the following syntax to start |compass| and set a configuration option:
 Options
 -------
 
-You can set the following configuration options using the command line:
+You can set these options when using the command line:
+
+.. option:: --autoUpdates
+  
+   Enable automatic updates.
+  
+.. option:: --enableFeedbackPanel
+  
+   Enable the feedback panel.
+   
+.. option:: --enableMaps
+ 
+   Enable geographic visualizations.
+
+.. option:: --exportConnections
+    
+   Export :ref:`favorite connections 
```

Comment:
The import / export page is pretty long. Can this link directly to export rather than the top of the page?

Reply:
I've updated these links so that the import-related option points to the Import Connections page and the export-related option points to the Export Connections page.